scripts/cmake-build-mac.sh

```diff
@@ -23,12 +23,21 @@ _init
 
 ##### ### # Global definitions # ### ########################################
 ##### ### # Mac Qt # ### ####################################################
-MAC_QT_DIR=${QT_INSTALLATION_PATH}/${QT_VERSION_MAC}/clang_64
+if [[ -z "${MAC_QT_DIR}" ]]; then
+    MAC_QT_DIR=${QT_INSTALLATION_PATH}/${QT_VERSION_MAC}/clang_64
+else
+    MAC_QT_DIR="${MAC_QT_DIR}"
```

Comment:
If $MAC_QT_DIR is not empty, it's already set. So why set it again?

Reply:
Fixed in 89b74e4b138af412b82066b3e08a2d31f27a2ec5